lib/assessment.js

```diff
@@ -5,7 +5,13 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
-
+  minimum = arguments[0];
+  for (var i =0; i < arguements.length; i++) {
```

Comment:
You've got a typo in the condition expression here. A good strategy for finding typos is to highlight the variable name in sublime. All other cases of that variable name nearby will have a rectangle drawn around them, provided they're spelled the same. The one without the rectangle around it is the typo!